app/_hub/kong-inc/exit-transformer/index.md

```diff
@@ -191,26 +276,81 @@ end
 
       return status, body, headers
```

Comment:
My bad on this one (I think?). It should read `return status, new_body, headers`